python/pyspark/sql/tests/connect/test_connect_basic.py

```diff
@@ -159,6 +159,19 @@ def spark_connect_clean_up_test_data(cls):
 
 
 class SparkConnectBasicTests(SparkConnectSQLTestCase):
+    def test_recursion_handling_for_plan_logging(self):
```

Comment:
Hi, @grundprinzip and @HyukjinKwon .

This test case seems to fail with Python 3.11. SPARK-45987 is filed for Python 3.11 failure.